tests/conformance/test_conformance.py

```diff
@@ -0,0 +1,729 @@
+# read or just a metadata get).
+
+method_mapping = {
+    "storage.buckets.delete": [bucket_delete],  # S1 start
+    "storage.buckets.get": [
+        client_get_bucket,
+        bucket_reload,
+        client_lookup_bucket,
+        bucket_exists,
+    ],
+    "storage.buckets.getIamPolicy": [bucket_get_iam_policy],
+    "storage.buckets.insert": [client_create_bucket, bucket_create],
+    "storage.buckets.list": [client_list_buckets],
+    "storage.buckets.lockRententionPolicy": [],
+    "storage.buckets.testIamPermission": [bucket_test_iam_permissions],
+    "storage.notifications.delete": [notification_delete],
+    "storage.notifications.get": [
+        bucket_get_notification,
+        notification_exists,
+        notification_reload,
+    ],
+    "storage.notifications.list": [bucket_list_notifications],
+    "storage.objects.get": [
+        bucket_get_blob,
+        blob_exists,
+        client_download_blob_to_file,
+        blob_download_to_filename,
+        blob_download_as_bytes,
+        blob_download_as_text,
+        blobreader_read,
+    ],
+    "storage.objects.list": [
+        client_list_blobs,
+        bucket_list_blobs,
+        bucket_delete,
+    ],  # S1 end
+    "storage.buckets.patch": [bucket_patch],  # S2 start
+    "storage.buckets.setIamPolicy": [bucket_set_iam_policy],
+    "storage.buckets.update": [bucket_update],
+    "storage.objects.compose": [blob_compose],
+    "storage.objects.copy": [bucket_copy_blob, bucket_rename_blob],
+    "storage.objects.delete": [
+        bucket_delete_blob,
+        bucket_delete_blobs,
+        bucket_delete,
+        blob_delete,
+        bucket_rename_blob,
+    ],
+    "storage.objects.insert": [
+        blob_upload_from_string,
+        blob_upload_from_file,
+        blob_upload_from_filename,
+        blobwriter_write,
+        blob_create_resumable_upload_session,
+    ],
+    "storage.objects.patch": [blob_patch],
+    "storage.objects.rewrite": [blob_rewrite, blob_update_storage_class],
+    "storage.objects.update": [blob_update],  # S2 end
+}
+
+
+########################################################################################################################################
+### Pytest Fixtures to Populate Resources ##############################################################################################
+########################################################################################################################################
+
+
+@pytest.fixture
+def client():
+    host = os.environ.get(_STORAGE_EMULATOR_ENV_VAR)
+    client = storage.Client(
+        project=_CONF_TEST_PROJECT_ID,
+        credentials=AnonymousCredentials(),
+        client_options={"api_endpoint": host},
+    )
+    return client
+
+
+@pytest.fixture
+def bucket(client):
+    bucket = client.bucket(uuid.uuid4().hex)
+    client.create_bucket(bucket)
+    yield bucket
+    try:
+        bucket.delete(force=True)
+    except Exception:
```

Comment:
As you've identified, these would ideally be narrower so we can find out if there is a problem with clean-up, but I agree it's okay to have this as-is until the emulator is improved.